### ISSUE TITLE
add Cancel to Create dialog to re-enable "Search or create file"  & "Library" in Files tab

### ISSUE THREAD

**Issue by DrXyzzy** [sagemathinc/cocalc#3543]
tl;dr: File search and Library browse are not accessible in a project after clicking Create until browser is refreshed or file is created.

1. Open Files tab in a project.
1. Click Create. Text input to left of Create button, "Search or create file", is disabled.
1. Now suppose you decide to search for some file before choosing a name for the file to be created, or decide not to create a file at all. "Search or create file" remains disabled if you change folders, open another file and come back, or another project, or another browser tab. It is re-enabled if you refresh the browser tab.

Consider either:
* add a Cancel button when Create has been selected
* don't disable "Search or create file" & Library

**Comment by williamstein** [comment]
> * Cancel button when Create has been selected

There is one, but it is obviously way too hard to find (it's also called "Close") - it is in the lower right.

> * don't disable "Search or create file" & Library

Agreed; there is no reason to disable that, which I can think of...
